hydro/IO/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# build the orchestrator static library
+add_library(hydro_netcdf_layer STATIC
```

Comment:
Target is missing includes / "linking" to netCDF-Fortran. My setup did not get past this without specifically including netCDF here - not sure if there is a specific way netCDF was provided in current testing of this build.

Reply:
Added NetCDF include directories